### ISSUE TITLE
Addition of (redesigned earlier) new Web Mockups

### ISSUE THREAD

**Issue by nazmicancalik** [bounswe/bounswe2018group9#121]
Due to the problems with the proto.io (which we used to draw mockups), we weren't able to get and add the redisigned web mockups to our wiki page. Finally, somehow we have managed to get the screenshots of those web mockups pages. And they are added with a delay.

[Go to web mockups](https://github.com/bounswe/bounswe2018group9/wiki/Web-Mockups)

Ps: This issue is opened for information purposes.
@mericturan

**Comment by mericturan** [comment]
The new design for the web mock-ups seems much nicer. I have two things to say though:

1. When did the problems start to occur with proto.io?
2. You still have your old web mock-up design in page [Mockups(Both)](https://github.com/bounswe/bounswe2018group9/wiki/Mockups(Both)).

**Comment by nazmicancalik** [comment]
* Our trial ended after 2 weeks from the time we started doing web mockups. We created these new mockups back then however couldn't take them from the system. After a while we were able to get the screenshots. So we put them with a little delay but the original work has been done at its time.

* I will now swap the old mockups with the new ones in the **Mockups (Both)** page also, thanks for reminder.
